### ISSUE TITLE
Add accessibility blocks to component documentation

### ISSUE THREAD

**Issue by bhushan-ebi** [visual-framework/vf-core#1980]
Many components lack accessibility related details in the documentation. This needs to be added where missing and updated where needed.

**Comment by Adedoyinebi** [comment]
As discussed during previous sprint planning meetings. The accessibility checks will be done as part of the review tasks which have been created:
- https://github.com/visual-framework/vf-core/issues/2049
- https://github.com/visual-framework/vf-core/issues/2050
- https://github.com/visual-framework/vf-core/issues/2051 

For other elements accessibility block have been added to some elements based on previous reviews stating that "This component targets WCAG 2.1 AA accessibility", accessibility checks should be added to the process of any future enhancements

Closing this task.
